Create 202010-helm-stable-eol.md (#490)
giantswarm/releases: announcements/202010-helm-stable-eol.md

:information_source: On Nov 13, 2020, the upstream Helm `stable` repo will be obsolete. Consequently, so will the "Helm Stable" Catalog in our Web UI (Happa). (See: [Helm Deprecation Timeline](https://github.com/helm/charts/blob/master/README.md#deprecation-timeline))

> In case you are using apps installed from Helm Stable, we recommend to create your own App Catalog ([docs](https://docs.giantswarm.io/reference/kubectl-gs/template-appcatalog/)) and follow the apps' respective upstream chart. As always, feel free to reach out to your Solution Engineer for assistance.

Author: Is there a more "step by step" doc for creating your own App Catalog?

Reviewer: yeah this would need a better guide... letting customers create their own catalog is not really something we wanted to actively encourage yet right?

can we instead say something like.. "speak with your Solution Engineer for an alternative", and not mention making their own catalog?

Reviewer 2: Is this https://intranet.giantswarm.io/docs/dev-and-releng/app-developer-processes/creating_app_catalog/ still legit to create an appcatalog?

Reviewer 3: We've had some customers create their own catalogs. Mainly for bootstrapping their own CI/CD processes.

But its not something we're actively pushing and @oponder I like your suggestion of talk to your SE.

@tomahawk28 Yes those are the docs for adding a new catalog to `opsctl ensure appcatalogs`. But customers can still create their own appcatalog CRs.

Author: Thanks! Yes, we are not actively improving the experience of creating your own Catalog.

But what action do we (Giant Swarm) encourage customers to take? In my mind, it's "Create your own App Catalog. The docs are not the best, so please work with your SE." Is this what you had in mind as well? Or when they speak with their SE, what will their SE tell them?

Reviewer: SE will tell them about making their own catalog :P ... but at least we wouldn't be actively promoting it to people that way.

> In case you are using apps installed from Helm Stable, we recommend to create your own App Catalog ([docs](https://docs.giantswarm.io/reference/kubectl-gs/template-appcatalog/)) and follow the apps' respective upstream chart. As always, feel free to reach out to your Solution Engineer for assistance.

Author: ```suggestion
In case you are using apps installed from the Helm Stable Catalog, please move to following the apps' respective upstream charts or reach out to your Solution Engineer for alternatives.
```

@giantswarm/team-batman what about this?

Reviewer 3: @cokiengchiara Yes, that LGTM 👍
